DigitalOffice/Services/AuthenticationService/Validators/UserLoginValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.AuthenticationService.Models;
+
+namespace LT.DigitalOffice.AuthenticationService.Validators
+{
+    public class UserLoginValidator : AbstractValidator<UserCredentialsRequest>
+    {
+        public UserLoginValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .MaximumLength(164)
+                .WithMessage("Email is too long.")
+                .EmailAddress()
+                .WithMessage("Email is invalid.");
+
+            RuleFor(user => user.Password)
+                .NotEmpty()
+                .WithMessage("Password is empty.")
+                .MinimumLength(6)
```

Comment:
In UserCreateRequestValidator password may be less than 6 in length

Reply:
Why may be less than 6 in length? Do you think that your password should look like: 1234? All other systems validate password on the lenght (minimum 6 chars)

Reply:
In the validator when creating a user in UserService, the password can be of any length. For example, a user creates an account with a password of length 3 and then trying to log in, he receives an error that the password is too short.

Reply:
If look with this point of view, I agree.

Reply:
Fixed